ui/lib/kv/addon/components/page/secret/metadata/details.hbs

```diff
@@ -57,7 +48,7 @@
   Custom metadata
 </h2>
 <div class="box is-fullwidth is-sideless is-paddingless is-marginless" data-test-kv-custom-metadata-section>
-  {{#if (or @metadata.canReadMetadata @secret.canReadData)}}
+  {{#if @canReadCustomMetadata}}
```

Comment:
Could we instead check for `this.customMetadata`? Since the read already happened I'm not sure why we need this capability

Reply:
Sure - I was hesitant to change the workflow too much